src/LayerEditor/ui/panels/regions_panel.py

```diff
@@ -463,12 +446,18 @@ def _update_region_content(self):
 
 
     # ======= Internal signal handlers
-    def _combo_set_region(self, region_id):
+    def _combo_set_region(self, combo_id):
         """
         Handle change in region combo box.
         """
-        self._region_id = region_id
-        self.layer_heads.select_regions({self.layer_id: region_id})
+        for reg_id, idx in self._combo_id_to_idx.items():
```

Comment:
Just pass reg_id as before.

Reply:
Not possible. This is slot function, which handles selection in ComboBox. But was severely changed due to previous issues.

Reply:
https://github.com/GeoMop/GeoMop/blob/4f76b008e7f38f1411f210406206f120dd7ed7a0/src/LayerEditor/ui/panels/region_panel_tab.py#L206-L217